cmd/kubevpn/cmds/connect.go

```diff
@@ -87,6 +90,7 @@ func CmdConnect(f cmdutil.Factory) *cobra.Command {
 			} else {
 				resp, err := cli.Connect(cmd.Context(), req)
 				if err != nil {
+					err = errors.Wrap(err, "cli.Connect(cmd.Context(), req): ")
```

Comment:
how about just convert "cli.Connect(cmd.Context(), req): " --> "failed to connect to cluster" ?